_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -74834,8 +74833,7 @@
 	heat_proof = 1;
 	name = "Supermatter Chamber"
 	},
-/obj/effect/mapping_helpers/airlock/access/any/engineering/atmos,
-/obj/effect/mapping_helpers/airlock/access/any/engineering/general,
+/obj/effect/mapping_helpers/airlock/access/all/engineering/atmos,
```

Comment:
this change needs undoing, we want the two `any` helpers here, not two `all` helpers